src/Files.App/UserControls/OngoingTasksFlyout.xaml.cs

```diff
@@ -37,14 +34,7 @@ private async void Button_Click_1(object sender, RoutedEventArgs e)
 
 		private void DismissAllBannersButton_Click(object sender, RoutedEventArgs e)
 		{
-			for (int i = OngoingTasksViewModel.StatusBannersSource.Count - 1; i >= 0; i--)
-			{
-				var itemToDismiss = OngoingTasksViewModel.StatusBannersSource[i];
-				if (!itemToDismiss.IsProgressing)
-				{
-					OngoingTasksViewModel.CloseBanner(itemToDismiss);
-				}
-			}
+			OngoingTasksViewModel.CloseAllBanner();
```

Comment:
While you're in here, would it be a good idea to rename it to "ViewModel"?

Reply:
OngoingTasksViewModel cannot be renamed to "ViewModel" because it is referenced from AddressToolbar which has a ToolbarViewModel named "ViewModel".

Reply:
I was a little mistaken. I have renamed it.